docker/build-venice-docker-images.sh

```diff
@@ -2,21 +2,30 @@ cd ../
 ./gradlew shadowJar
 cd docker
```

Comment:
You haven't changed this but I'm just curious... is this script expected to be run from the root of the repo, i.e. `./docker/build-venice-docker-images.sh`?

Reply:
I have enhanced the script in order to let users run it from anywhere (my bash fu is very weak, but I think that my change should work)